ios/FluentUI.Demo/FluentUI.Demo/Demos/DateTimePickerDemoController.swift

```diff
@@ -6,6 +6,43 @@
 import FluentUI
 import UIKit
 
+public final class DemoBarButtonItems: NSObject {
```

Comment:
don't need to be final or public. because this is about datetimepickerdemocontroller, i would recommend to put this class define in the bottom of the file

Reply:
This supporting class has been completely removed

Reply:
Removed the whole 'DemoBarButtonItems' class